templates/main.template

```diff
@@ -100,165 +126,239 @@ Parameters:
       InstanceTypeDatabase: db.m4.large
     ap-southeast-1:
-      AMZNLINUXHVM: ami-05b3bcf7f311194b3
+      AMZNLINUXHVM: ami-03097abf0db1cdff2
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     ap-southeast-2:
-      AMZNLINUXHVM: ami-02fd0b06f06d93dfc
+      AMZNLINUXHVM: ami-05067171f4230ac41
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     ca-central-1:
-      AMZNLINUXHVM: ami-07423fb63ea0a0930
+      AMZNLINUXHVM: ami-07ab3281411d31d04
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     eu-central-1:
-      AMZNLINUXHVM: ami-0cfbf4f6db41068ac
+      AMZNLINUXHVM: ami-03a71cec707bfc3d7
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     eu-west-1:
-      AMZNLINUXHVM: ami-08935252a36e25f85
+      AMZNLINUXHVM: ami-03c242f4af81b2365
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     eu-west-2:
-      AMZNLINUXHVM: ami-01419b804382064e4
+      AMZNLINUXHVM: ami-05663d374a152d239
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     sa-east-1:
-      AMZNLINUXHVM: ami-05145e0b28ad8e0b2
+      AMZNLINUXHVM: ami-0eb2a191bf5e40e10
       InstanceType: m4.large
       InstanceTypeDatabase: db.m3.large
     us-east-1:
-      AMZNLINUXHVM: ami-0080e4c5bc078760e
+      AMZNLINUXHVM: ami-0756fbca465a59a30
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     us-east-2:
-      AMZNLINUXHVM: ami-0cd3dfa4e37921605
+      AMZNLINUXHVM: ami-04768381bf606e2b3
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     us-gov-west-1:
       AMZNLINUXHVM: ami-ffa61d9e
       InstanceType: m4.large
       InstanceTypeDatabase: db.m3.large
     us-west-1:
-      AMZNLINUXHVM: ami-0ec6517f6edbf8044
+      AMZNLINUXHVM: ami-063dd30adbb186909
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
     us-west-2:
-      AMZNLINUXHVM: ami-01e24be29428c15b2
+      AMZNLINUXHVM: ami-07a0c6e669965bb7c
       InstanceType: m4.large
       InstanceTypeDatabase: db.m4.large
 Conditions:
   GovCloudCondition:
     !Equals
     - !Ref AWS::Region
     - us-gov-west-1
-  LoadConfigRulesTemplate:
-    !Equals
-    - !Ref pSupportsConfig
-    - 'true'
   LaunchAsDedicatedInstance:
     !Equals
     - !Ref pVPCTenancy
     - dedicated
 Resources:
+  IAMPasswordPolicy:
+    Type: 'Custom::IAMPolicyResource'
+    Properties:
+      ServiceToken: !GetAtt IAMPasswordPolicyResource.Arn
+      Region: !Ref "AWS::Region"
+
+  IAMPasswordPolicyResource:
+    Type: "AWS::Lambda::Function"
+    Properties:
+      Runtime: python2.7
+      Handler: index.lambda_handler
+      MemorySize: 128
+      Timeout: 30
+      Role: !GetAtt IAMPasswordPolicyResourceExecutionRole.Arn
+      Code:
+        ZipFile: !Sub |
+          import boto3
+          from botocore.exceptions import ClientError
+          import json
+          import cfnresponse
+
+          iam = boto3.client("iam")
```

Comment:
`iam = boto3.client("iam", region={AWS::Region})`

I'm not 100% certain that will work out the box,but I think you get the point.

Reply:
I am not sure adding the region is needed? I have deployed the template in multiple regions and have not had issues with the password policy...trying to add the region here throws an error.